jbpm-services/jbpm-executor/src/test/java/org/jbpm/executor/BasicExecutorBaseTest.java

```diff
@@ -631,16 +631,18 @@ public void testPrioritizedJobsExecutionInvalidProrities() throws InterruptedExc
         CommandContext ctxCMD = new CommandContext();
         ctxCMD.setData("businessKey", "low priority");
         ctxCMD.setData("priority", -1);
+        ctxCMD.setData("delay", 10L);
         
         Date futureDate = new Date(System.currentTimeMillis() + EXTRA_TIME);
 
-        executorService.scheduleRequest("org.jbpm.executor.commands.PrintOutCommand", futureDate, ctxCMD);
+        executorService.scheduleRequest("org.jbpm.executor.commands.DelayedPrintOutCommand", futureDate, ctxCMD);
```

Comment:
This will just prolong the execution of the command by including the `Thread.sleep()` in it. I think we discussed change on the line 664 that we need `>=` there, didn't we?

Reply:
Yes, we discussed "greater and equals" as a possible solution; my point is that if the command lasts several milliseconds (with the delay we can control that), then, we may have a strict greater. Also I've noticed that there were no tests for DelayPrintOutCommand. But, if you prefer, we can just keep the change as commented previously.

Reply:
Well, if i understand it correctly, the duration of the command doesn't influence the `time` field of the `RequestInfo` object. This field is updated only when the request is obtained from DB to be run. See here:
https://github.com/kiegroup/jbpm/blob/05a632fc1c11608a89dbce5f63532af04b38fb97/jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/jpa/ExecutorQueryServiceImpl.java#L283-L291

It is not updated afterwards, at least I haven't found any occurrence of it. But I may be missing something, of course.

Reply:
You're right; equals comparison is therefore needed anyway. Keep it simple with just that change. Thanks!